com.unity.netcode.gameobjects/Tests/Runtime/NetworkTransform/NetworkTransformTests.cs

```diff
@@ -292,6 +292,10 @@ public override IEnumerator Setup()
             m_ClientSideObject.SetActive(false);
         }
 
+        /// <summary>
+        /// Disabling until snapshot is finished.
+        /// </summary>
+        [Ignore("Snapshot transition")]
```

Comment:
I thought `1.1` had snapshot disabled. How is this needed?

Reply:
I'm actually not sure. Maybe this test was breaking and Noel decided to defer that after we have snapshots, IDK.

Reply:
Is it possible that the decision to backport was in error and we simply don't need this ? e.g. If CI is reliably passing without it, maybe don't backport ?

Reply:
yes, I think I agree with you here — if the other PR #1536 just simply passes and merges, I'll close this one off and cancel the backport as it'd not be needed.